### DISCUSSION TITLE
entrypoint executável

### DISCUSSION THREAD

**Discussion by glatzmarcus** [community/community#37910] [Actions]
Oi, estava com uma certa dificuldade de transformar em exuctável o arquivo entrypoint, quando estava criando uma ação para um container do docker. 

No código: $ git update-index --chmod=+x entrypoint.sh

deveria ser: $ git update-index --add --chmod=+x entrypoint.sh

Dessa forma, como não é um simples update pois não tinha adicinado ainda, funcionou aqui para mim.

**Comment by ossanna16** [comment]
Hi there @glatzmarcus and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
